pkg/analytics/analytics.go

```diff
@@ -0,0 +1,105 @@
+package analytics
+
+import (
+	"encoding/json"
+	"os"
+	"regexp"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+// Variables that we'll set @ build time
+var (
+	Tier    = "oss"
+	Version = "v0.0.0"
+)
+
+const (
+	app              = "cli"
+	trackEventURL    = "https://app.pendo.io/data/track"
+	trackEventSecret = "bf6ab33e-cd70-46e7-4b77-279f54cac447"
+)
+
+type analyticsRequestBody struct {
+	Tp         string            `json:"type"`
+	Event      string            `json:"event"`
+	VisitorID  string            `json:"visitorId"`
+	Timestamp  int64             `json:"timestamp"`
+	Properties map[string]string `json:"properties"`
```

Comment:
BTW, what is interesting, in case of an invalid request, Pendo returns an object with fields like  "timestamp field is required". Otherwise it returns nothing.